binaries/scheduler/main.go

```diff
@@ -1,33 +1,61 @@
 package main
 
 import (
+	"flag"
+	"fmt"
 	"log"
+	"strconv"
+	"strings"
 	"sync"
 
 	"github.com/apache/thrift/lib/go/thrift"
-	//clusterdef "github.com/scootdev/scoot/cloud/cluster"
+	clusterdef "github.com/scootdev/scoot/cloud/cluster"
 	clusterimpl "github.com/scootdev/scoot/cloud/cluster/memory"
 	"github.com/scootdev/scoot/saga"
 	"github.com/scootdev/scoot/sched/distributor"
 	queueimpl "github.com/scootdev/scoot/sched/queue/memory"
 	"github.com/scootdev/scoot/sched/scheduler"
+	"github.com/scootdev/scoot/sched/worker"
 	//"github.com/scootdev/scoot/sched/worker"
 
 	"github.com/scootdev/scoot/sched/worker/fake"
+	"github.com/scootdev/scoot/sched/worker/rpc"
 	// "github.com/scootdev/scoot/sched/worker/rpc"
 	"github.com/scootdev/scoot/scootapi/server"
 )
 
+var addr = flag.String("addr", "localhost:9090", "Bind address for api server.")
+var workers = flag.String("workers", "", "Comma separated list of workers (host:port,...)|NUM:mem.")
+
 func main() {
 	log.Println("Starting Cloud Scoot API Server & Scheduler")
-	addr := "localhost:9090"
+
+	flag.Parse()
+	workersList := strings.Split(*workers, ",")
+	workerNodes := []clusterdef.Node{}
```

Comment:
Added a TODO. This is similar to what we discussed for scootfs, so we'll probably want to stick to one format, and json seems fine.